cmd/metal-api/internal/service/integration_test.go

```diff
@@ -1,3 +1,4 @@
+//nolint:unused
```

Comment:
I think this arises due to the `// +build integration` tag? It's also possible to run goloangci-lint with `--build-tags`, which potentially resolves this problem.